gk/main.c

```diff
@@ -429,17 +429,19 @@ gk_process_granted(struct flow_entry *fe, struct ipacket *packet,
 		priority = PRIORITY_RENEW_CAP;
 	}
 
+	struct grantor_entry *grantor = choose_grantor_per_flow(fe);
```

Comment:
Style: declare `struct grantor_entry *grantor;` at the beginning of the function.